packages/table/src/cell/_common.scss

```diff
@@ -18,12 +18,15 @@ $dark-cell-background-color: $dark-gray3 !default;
 $dark-cell-border-color: $pt-dark-divider-black !default;
 $dark-cell-text-color: $pt-dark-text-color !default;
 
+$text-fade-in-transition: color $pt-transition-duration * 3;
```

Comment:
hrmm i take it back. make the variable be just the `-duration` and add ` !default` to the end.